### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'text'

### ISSUE THREAD

**Issue by johnthomas00** [juliomalegria/python-craigslist#16]
Love this Python application, thanks.  It has been working great, but today, I started getting this error.  Any suggestions:

```
Traceback (most recent call last):
  File "./get_search.py", line 28, in <module>
    for x in gems.get_results(sort_by='newest', geotagged=True):
  File "/home/user47/craigslist_app/lib/python3.4/site-packages/craigslist/__init__.py", line 160, in get_results
    p_text = row.find('span', {'class': 'p'}).text
AttributeError: 'NoneType' object has no attribute 'text'
```

**Comment by johnthomas00** [comment]
As a temporary work around, I replaced
    p_text = row.find('span', {'class': 'p'}).text
with
    if row.find('span', {'class': 'p'}):
        p_text = row.find('span', {'class': 'p'}).text
but not sure that works yet as I only run this every few hours.  I will report back later.

**Comment by johnthomas00** [comment]
As a hack/solution, I replaced line 160:
`p_text = row.find('span', {'class': 'p'}).text`
with
```
p_text = ''
    if row.find('span', {'class': 'p'}):
        p_text = row.find('span', {'class': 'p'}).text
```

**Comment by juliomalegria** [comment]
Thanks for the feedback John! I've fixed it in ebd99b5.
This will be included in the next release :)
